### PR TITLE
#2 and #3 Electron Packager/installer for windows and mac

### PR DESCRIPTION
**Pull request by adarsh1021**
1. Install the app by running (Now this also installs the required electron packager and installers) :  
npm install

2. To build run : 
npm run package-win
npm run package-mac

3.  To create installer files run : 
npm run create-installer-win
npm run create-installer-mac

I tested on windows. Working well.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,5 +19,11 @@
     "mongodb": "^2.2.28",
     "path": "^0.12.7",
     "url": "^0.11.0"
+  },
+  "devDependencies": {
+    "electron": "^1.4.3",
```
Comment (championswimmer) [inline package.json]: already in dependencies. remove electron from dependencies, and update the version to 1.6.6 here

**Review by championswimmer — CHANGES_REQUESTED** [review]

**Comment by piyush0** [comment]
@adarsh1021 Since build for mac has already been solved in #4. Remove all changes that are associated with mac so as to avoid any confusion

**Comment by piyush0** [comment]
Did you run it after building ? I tested it on windows 64 bit. The build was successful but I am getting this error on running Snipper 
image

**Comment by adarsh1021** [comment]
I did not run the application. Sorry about that. Since I had not made any changes to the main source code I thought it would be fine.
The issue was a small filename error which I have renamed now. 
On running on my system (win10 64bit) I am getting this error, but I think this is because I have not setup the database. 
first
On clicking OK I get this, so should work fine on your system. 
second

**Comment by championswimmer** [comment]
conflicts need resolution

**Pull request closed without merging**